Automattic/wp-calypso
# Site Settings: add Survey Q1 to the JP Disconnect UX

This PR is part of the ⭐️ new Jetpack Disconnect Flow ⭐️ .
**Depends on**: https://github.com/Automattic/wp-calypso/pull/17691 

**Included changes:**

- add the first survey Question to the JP Disconnect Flow. 
- add placeholder transition to the follow-up screen. Note: the transition type was changed to the 'horizontal' one, where the follow-up action will be displayed in a clean component view.

**View:**
`/settings/disconnect-site/:site`, where `:site` is a Jetpack or non-Atomic site.
For other sites the Flow redirects automatically to `/settings/general:site`.

**To test**:

- Checkout this branch, or use calypso.live:
https://calypso.live/?branch=update/site-settings-disconnect-site-survey

Verify that:
- survey options are visible (see below)
- upon selection, the Main survey component is cleared and shows a placeholder for subsequent actions (see below)
 
**Visuals**:

Example view for a Free plan (2 options): 

![screen shot 2017-09-30 at 01 36 34](https://user-images.githubusercontent.com/13561163/31040486-c4475fa4-a57f-11e7-9177-df8f1fd62bd2.png)

For other Plans we should see an additional `This plan is too expensive` option.

Follow-up action cleans the view:

![screen shot 2017-09-30 at 01 57 00](https://user-images.githubusercontent.com/13561163/31040770-69d6605c-a583-11e7-9fa7-080629990700.png)

The above is a placeholder for the follow-up action or question.

**Note for reviewers:**
- proper survey data storage will be implemented in a follow-up (this patch uses the state to store Q1 selection)
- Placeholder used for initial rendering (empty Redux store) will be adapted to the Disconnect flow in the follow-up
- *Back* button redirects to `/settings/manage-connection/:site`
- *Skip Survey* redirects to `/settings/manage-connection/:site`
